docs/source/changelog.rst

```diff
@@ -6,6 +6,7 @@ Changelog
 
 - Resolve issue where the optional ``--noauth_local_webserver`` command line argument would not be propagated during the authentication process. (:issue:`35`)
 - Drop support for Python 3.4 (:issue:`40`)
+- Add support for users to provide a table schema in ``to_gbq`` call instead of letting module to infer the schema from ``DataFrame.dtypes`` (:issue:`46`)
```

Comment:
Add support for a passed schema in :func:`to_gbq` instead inferring the schema from the passed ``DataFrame`` with ``DataFrame.dtypes``